source/extensions/omni.isaac.lab/omni/isaac/lab/sensors/rtx_lidar/rtx_lidar_cfg.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2022-2024, The Isaac Lab Project Developers.
+# All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+from __future__ import annotations
+
+from dataclasses import MISSING
+from typing import Literal
+
+from omni.isaac.lab.sensors import SensorBaseCfg
+from omni.isaac.lab.sim import LidarCfg
+from omni.isaac.lab.utils import configclass
+
+from .rtx_lidar import RtxLidar
+
+
+@configclass
+class RtxLidarCfg(SensorBaseCfg):
+    """Configuration for the RtxLidar sensor."""
+
+    @configclass
+    class OffsetCfg:
+        """The offset pose of the sensor's frame from the sensor's parent frame."""
+
+        pos: tuple[float, float, float] = (0.0, 0.0, 1.0)
```

Comment:
why (0, 0, 1) instead of (0, 0, 0)?

Reply:
good catch I was using that during manual testing